Tests/PageObject/RegisterPage.py

```diff
@@ -0,0 +1,48 @@
+import faker
+
+from Tests.PageObject.BasePage import BasePage
+from Tests.PageObject.locators import RegisterPageLocators
+
+fake = faker.Faker()
+password = fake.password()
+first_name = fake.first_name()
+last_name = fake.last_name()
+
+
+class RegisterPage(BasePage):
+    def guest_can_go_to_register_page(self):
+        self.should_be_customer_form()
+
+    def should_be_customer_form(self):
+        assert self.is_element_present(*RegisterPageLocators.RIGHT_PANEL), "Right panel is not present"
```

Comment:
each element should be described like a separete function

Reply:
done